src/jgikbase/idmapping/config.py

```diff
@@ -165,43 +180,64 @@ def _get_cfg_from_env(self) -> Path:
             return Path(os.environ[self.ENV_VAR_IDMAPPING])
         if os.environ.get(self.ENV_VAR_KB_DEP):
             return Path(os.environ[self.ENV_VAR_KB_DEP])
-        raise IDMappingConfigError('Could not find deployment configuration file from either ' +
-                                   'permitted environment variable: {}, {}'.format(
-                                       self.ENV_VAR_IDMAPPING, self.ENV_VAR_KB_DEP))
+        raise IDMappingConfigError(
+            "Could not find deployment configuration file from either "
+            + "permitted environment variable: {}, {}".format(
+                self.ENV_VAR_IDMAPPING, self.ENV_VAR_KB_DEP
+            )
+        )
 
-    def _get_string(self, param_name: str, config: Dict[str, str], raise_on_err: bool=True
-                    ) -> Optional[str]:
+    def _get_string(
+        self, param_name: str, config: Dict[str, str], raise_on_err: bool = True
+    ) -> Optional[str]:
         s = config.get(param_name)
         if s and s.strip():
             return s.strip()
         elif raise_on_err:
             raise IDMappingConfigError(
-                'Required parameter {} not provided in configuration file {}, section {}'.format(
-                    param_name, config[self._TEMP_KEY_CFG_FILE], self.CFG_SEC))
+                "Required parameter {} not provided in configuration file {}, section {}".format(
+                    param_name, config[self._TEMP_KEY_CFG_FILE], self.CFG_SEC
+                )
+            )
         else:
             return None
 
-    def _get_authsource_ids(self, param_name: str, config: Dict[str, str]) -> Set[AuthsourceID]:
+    def _get_authsource_ids(
+        self, param_name: str, config: Dict[str, str]
+    ) -> Set[AuthsourceID]:
         s = self._get_string(param_name, config, False)
         ret: Set[AuthsourceID] = set()
         if not s:
             return ret
-        ids = s.split(',')
+        ids = s.split(",")
         for id_ in ids:
             try:
                 ret.add(AuthsourceID(id_.strip()))
             except MissingParameterError as e:
                 raise IDMappingConfigError(
-                    ('Parameter {} in configuration file {}, section {}, has whitespace-only entry'
-                     ).format(param_name, config[self._TEMP_KEY_CFG_FILE], self.CFG_SEC, str(e))
-                    ) from e
+                    (
+                        "Parameter {} in configuration file {}, section {}, has whitespace-only entry"
+                    ).format(
+                        param_name,
+                        config[self._TEMP_KEY_CFG_FILE],
+                        self.CFG_SEC,
+                        str(e),
```

Comment:
str(e) here is unused arg as well?

Reply:
Yep, seems that way. 

Flake8 must've changed a lot since the last time tests passed...

Reply:
ah, I seriously doubt the last time tests all passed.

Reply:
They did: https://github.com/jgi-kbase/IDMappingService/pull/126

Reply:
👍